test/e2e/main_test.go

```diff
@@ -46,7 +47,7 @@ var _ = BeforeSuite(func() {
 	nodeList := corev1.NodeList{}
 	err = framework.Global.Client.List(context.TODO(), &nodeList, &dynclient.ListOptions{})
 	Expect(err).ToNot(HaveOccurred())
-
+	reporter.BeforeSuiteDidRun()
```

Comment:
Instead of using it like this, I think we have to register the reporter at ginkgo with RunSpecsWithCustomReporters [1] for that we have to implement


```golang
type Reporter interface {
    SpecSuiteWillBegin(config config.GinkgoConfigType, summary *types.SuiteSummary)
    BeforeSuiteDidRun(setupSummary *types.SetupSummary)
    SpecWillRun(specSummary *types.SpecSummary)
    SpecDidComplete(specSummary *types.SpecSummary)
    AfterSuiteDidRun(setupSummary *types.SetupSummary)
    SpecSuiteDidEnd(summary *types.SuiteSummary)
}
```

[1]https://onsi.github.io/ginkgo/#writing-custom-reporters

Reply:
But we can do this in a follow up PR I think.

Reply:
i would prefer :)

Reply:
fair enough let's merge it ASAP is super nice addition!